### PR TITLE
Adding rssi

### PR DESCRIPTION
**Pull request by caguero**
# 🎉 New feature

## Summary

This PR adds the `rssi` value to the message sent between two robots using the comms infrastructure. This can be useful for estimating the range between the robots.

## Test it

* Run the tests.
* Run the example:

1. Launch Gazebo:
```
ign gazebo -v 4 <your_ws>/src/ign-gazebo/examples/worlds/rf_comms.sdf
```
2. Launch a subscriber:
```
ign topic -e -t addr2/rx
```
3. Launch a publisher:
```
cd <your_ws>/src/ign-gazebo/examples/standalone/comms
mkdir build && cd build
cmake ..
make
./publisher addr2
```
You should observe messages on the subscriber terminal with the new `rssi` field.

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [x] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸

### CODE REVIEW

File: src/systems/rf_comms/RFComms.cc [hunk 1]
```diff
@@ -15,6 +15,8 @@
  *
  */
 
+#include <ignition/msgs/dataframe.pb.h>
```
Comment (iche033) [inline src/systems/rf_comms/RFComms.cc:18]: move this include down to group it with the other ignition headers?

**Review by iche033 — APPROVED** [review]
looks good to me. Just one very minor comment

File: src/systems/rf_comms/RFComms.cc (hunk 1 above)
Comment (caguero) [reply src/systems/rf_comms/RFComms.cc:18]: > move this include down to group it with the other ignition headers?

It's a `.h`. If should be first in the list, right?

Comment (iche033) [reply src/systems/rf_comms/RFComms.cc:18]: oh just checked other places, looks like we do put these includes first. Look good then!

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**